lib/server.js

```diff
@@ -0,0 +1,112 @@
+const core = require('@actions/core')
+
+async function foundChannel(slackClient, channelName) {
+  const channels = await slackClient.conversations.list({
+    types: 'public_channel,private_channel'
+  }).channels
+
+  let result = channels.filter(chan => chan.name === channelName)
+  let foundedChannelId = undefined
```

Comment:
`found` est plus clair que `founded` , mais en vrai est-ce qu'on ne supprimerait pas cette notion 
```suggestion
  let channelId = undefined
```